### ISSUE TITLE
SettingsCard will not invoke command when scan mode is enabled.

### ISSUE THREAD

**Issue by dhoehna** [CommunityToolkit/Windows#395]
### Describe the bug

With narrator one and scan mode on, the `command` is not invoke when enter is pressed on the card.  

However, with narrator is on and scan mode in **off** pressing `enter` does invoke the command.

### Steps to reproduce

```text
1. Make a settings card in XAML.
2. Give it a command via the `command property`
3. Hook up the command with the `RelayCommand` atrtribute
4. Open the app.
5. Enable Narrator (default ctrl + win + enter)
6. Make sure scan is enabled (default CapsLock + Spacebar) Narrator should say "Scan"
7. Navigate to the settings card with tab.
8. Press enter.
```

### Expected behavior

The relay command is invoked.

### Code Platform

- [ ] UWP
- [X] WinAppSDK / WinUI 3
- [ ] Web Assembly (WASM)
- [ ] Android
- [ ] iOS
- [ ] MacOS
- [ ] Linux / GTK

### Windows Build Number

- [ ] Windows 10 1809 (Build 17763)
- [ ] Windows 10 1903 (Build 18362)
- [ ] Windows 10 1909 (Build 18363)
- [ ] Windows 10 2004 (Build 19041)
- [ ] Windows 10 20H2 (Build 19042)
- [ ] Windows 10 21H1 (Build 19043)
- [ ] Windows 10 21H2 (Build 19044)
- [ ] Windows 10 22H2 (Build 19045)
- [X] Windows 11 21H2 (Build 22000)
- [ ] Other (specify)

### App minimum and target SDK version

- [ ] Windows 10, version 1809 (Build 17763)
- [ ] Windows 10, version 1903 (Build 18362)
- [ ] Windows 10, version 1909 (Build 18363)
- [ ] Windows 10, version 2004 (Build 19041)
- [ ] Windows 10, version 2104 (Build 20348)
- [X] Windows 11, version 22H2 (Build 22000)
- [ ] Other (specify)

### Visual Studio Version

2022

### Visual Studio Build Number

17.9.3

### Device form factor

Desktop

### Help us help you

No, I'm unable to contribute a solution.

**Comment by niels9001** [comment]
Duplicate of #391
